src/components/Form/Form.js

```diff
@@ -1280,23 +1281,25 @@ class Form extends Component {
     }
 
     renderCourseRegistrationSubmission(){
+        console.log(this.state)
         if(this.props.registeredCourses){
 
             let currentStudentID = this.state.Student.Student.value;
             let registeredCourseForm = this.props.registeredCourses[currentStudentID];
             registeredCourseForm = registeredCourseForm[registeredCourseForm.length - 1];
-
             let currentStudentName = registeredCourseForm.display.student_name;
             let currentCourseTitle = registeredCourseForm.display.course_name;
+            let student=currentStudentName.slice(0,currentStudentName.indexOf(" -"));
```

Comment:
Don't do string manipulation here. Keep string manipulation centralized in the reducer (registrationReducer.js, for tutoring: line 332-ish, for class: line 223 ish).